lib/sos.ml

```diff
@@ -61,6 +61,20 @@ let show ?sandbox ?(action = `Skip) fmt =
       Ok ())
     fmt
 
+let cmd_error cmd err_msg status =
+  match (err_msg, status) with
+  | None, `Exited c | Some "", `Exited c ->
```

Comment:
You can simplify this into:
```suggestion
  | (None | Some ""), `Exited c ->
```

that of course apply to the other case below!

Reply:
Cool, that's a very nice pattern match. I like it :)